pages/common/cmake.md

```diff
@@ -0,0 +1,26 @@
+# cmake
+
+> CMake is a cross-platform build system generator.
+
+Make a project in the same directory as the source:
+
+```
+cmake
+make
```

Comment:
we don't use multi-line commands in tldr-pages. Maybe use `cmake && make` instead?